main.py

```diff
@@ -2,6 +2,39 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ("Western", "Action", "Dramma",)
+
+    for genre in genres:
+        Genre.objects.create(
+            name=genre
+        )
+
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+
+    for first_name, last_name in actors:
+        Actor.objects.create(
+            first_name=first_name,
+            last_name=last_name
+        )
+
+    Genre.objects.filter(name="Dramma").update(name="Drama")
+    Actor.objects.filter(first_name="George", last_name="Klooney").update(
+        last_name="Clooney")
```

Comment:
fix indentation here

Reply:
Can u pls be more acurate . if u talk about space after first_name i will fir. and if u talk about slice the line i can`t fix beacuse of PEP8

Reply:
or u are talking about  ')' ?